src/Microsoft.DotNet.ProjectModel/Compilation/LibraryExport.cs

```diff
@@ -35,9 +35,14 @@ public class LibraryExport
         public IEnumerable<LibraryAsset> CompilationAssemblies { get; }
 
         /// <summary>
+        /// Get a list of embedded resource files provided by this export.
+        /// </summary>
+        public IEnumerable<LibraryAsset> EmbeddedResources { get; }
+
+        /// <summary>
         /// Gets a list of fully-qualified paths to source code file references
         /// </summary>
-        public IEnumerable<string> SourceReferences { get; }
+        public IEnumerable<LibraryAsset> SourceReferences { get; }
```

Comment:
I think you need another type for SourceReference. It needs to take the language into account.